test/passes/pytorch/test_gptq.py

```diff
@@ -34,7 +34,7 @@ def test_gptq(tmp_path: Path, model_path: str, expected_model_type: str, group_s
         input_model = HfModelHandler(model_path=model_path)
     p = create_pass_from_dict(
         Gptq,
-        {"group_size": group_size, "lm_head": lm_head},
+        {"group_size": group_size, "lm_head": lm_head, "overrides": {"model.layers.0.self_attn.o_proj": {"bits": 8}}},
```

Comment:
From config file input, all the overrides dict is string. I don't see we convert str to int for this case. Does this take both 8 and "8"?